eng/dockerfile-templates/sdk/Dockerfile

```diff
@@ -15,10 +15,10 @@ RUN %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen update `
     && %windir%\Microsoft.NET\Framework\v4.0.30319\ngen update
 }}
 # Install NuGet CLI
-ENV NUGET_VERSION={{VARIABLES[cat("nuget|", OS_VERSION_NUMBER, "|version")]}}
+ENV NUGET_VERSION={{VARIABLES[cat("nuget|version")]}}
 RUN mkdir "%ProgramFiles%\NuGet\latest" `
     && curl -fSLo "%ProgramFiles%\NuGet\nuget.exe" https://dist.nuget.org/win-x86-commandline/v%NUGET_VERSION%/nuget.exe `
-    && curl -fSLo "%ProgramFiles%\NuGet\latest\nuget.exe" https://dist.nuget.org/win-x86-commandline/v{{VARIABLES["nuget|version"]}}/nuget.exe
+    && mklink "%ProgramFiles%\NuGet\latest\nuget.exe" "%ProgramFiles%\NuGet\nuget.exe"
```

Comment:
What do you think about modifying the [VerifyNuGetCli](https://github.com/microsoft/dotnet-framework-docker/blob/master/tests/Microsoft.DotNet.Framework.Docker.Tests/SdkOnlyImageTests.cs#L130) test to validate the link doesn't regress?  The core test logic could be easily modified to run against both paths.